app/file.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/mattermost/platform/model"
 	"github.com/mattermost/platform/utils"
 	s3 "github.com/minio/minio-go"
+	credentials "github.com/minio/minio-go/pkg/credentials"
```

Comment:
Jenkins doesn't like this import path. Does the version of minio-go in the vendor directory need to be updated?